multiqc_bcbio/bcbio.py

```diff
@@ -72,53 +72,53 @@ def __init__(self):
         self.metrics_stats_table()
 
         # General target stats
-        target_infos = list(self.find_log_files(config.sp['bcbio']['target']))
+        target_infos = list(self.find_log_files('bcbio/target'))
         if len(target_infos) == 1:
             add_project_info(yaml.load(target_infos[0]['f']))
 
         # Coverage plot
         # Only one section, so add to the intro
 
-        self.sections = list()
-        coverage_plot = self.bcbio_coverage_chart(config.sp['bcbio']['coverage'])
-        coverage_avg_plot = self.bcbio_coverage_avg_chart(config.sp['bcbio']['coverage_avg'])
+        coverage_plot = self.bcbio_coverage_chart('bcbio/coverage')
+        coverage_avg_plot = self.bcbio_coverage_avg_chart('bcbio/coverage_avg')
         qsignature_plot = None # disable plotting for now
-        #qsignature_plot = self.bcbio_qsignature_chart(config.sp['bcbio']['qsignature'])
-        umi_stats = self.bcbio_umi_stats(config.sp["bcbio"]["umi"])
-        if umi_stats:
-            self.sections.extend(umi_stats)
+        #qsignature_plot = self.bcbio_qsignature_chart('bcbio/qsignature')
+        for umi_section in self.bcbio_umi_stats("bcbio/umi"):
+            self.add_section(**umi_section)
 
-        viral_stats = self.get_viral_stats(config.sp["bcbio"]["viral"])
+        viral_stats = self.get_viral_stats("bcbio/viral")
         if viral_stats:
-            self.sections.append(viral_stats)
+            self.add_section(**viral_stats)
 
-        damage_stats = self.get_damage_stats(config.sp["bcbio"]["damage"])
+        damage_stats = self.get_damage_stats("bcbio/damage")
         if damage_stats:
-            self.sections.append(damage_stats)
+            self.add_section(**damage_stats)
 
         if mirna_stats.mirs:
-            self.sections.append({'name': 'miRNAs stats',
-                                  'anchor': 'bcbio-mirs',
-                                  'content': mirna_stats.mirs})
+            self.add_section(
+                name='miRNAs stats',
+                anchor='bcbio-mirs',
+                content=mirna_stats.mirs)
```

Comment:
If `mirna_stats.mirs` is just the returned plot data (looks like it is?) then please use `plot = mirna_stats.mirs` instead. Not much difference currently, but this new syntax is to enable better consistent formatting in the future for help text. Basically `content` = `description`+`help`+`plot`.